modules/backend/widgets/form/partials/_form_tabs.htm

```diff
@@ -12,13 +12,14 @@
     }
 ?>
 <div class="<?= $navCss ?>">
-    <ul class="nav nav-tabs">
+    <ul class="nav nav-tabs"  <?= $tabs->linkable ? 'data-linkable' : '' ?>>
         <?php $index = 0; foreach ($tabs as $name => $fields):
             $lazy = in_array($name, $tabs->lazy);
         ?>
             <li class="<?= ($index++ === 0) ? 'active' : '' ?> <?= $lazy ? 'tab-lazy' : '' ?>">
                 <a
                     href="#<?= $type . 'tab-' . $index ?>"
```

Comment:
How about just use the href and change it to `#<= $type . 'tab-' . str_slug($name); ?>`

Reply:
Does not it break something else? My solution is based on existing backend solution (`/backend/system/mailtemplates`) - why is `data-tab-url` used there?

Reply:
@Rike-cz I don't know. @daftspunk what is `data-tab-url` used for in `/backend/system/mailtemplates`?

Reply:
@daftspunk ...and why was `data-tab-url` used instead of `href`? Maybe there is reason which we do not see.

Reply:
I second what @LukeTowers  said, using at least 'tab-' as a prefix to data-tab-url name.  This way if some one wants to use an anchor hashtag with the same name as a tab, it doesn't conflict.

Reply:
@Rike-cz @LukeTowers 

Searching the entire codebase of OctoberCMS, the only reference to the data attribute tabUrl is in tabs.js.  All it appears to do is add the tab url hashtag to the address bar using history.replaceState.  

https://github.com/octobercms/october/blob/a68f3abf1c53cc702b6e748d927d95d25c59b31a/modules/system/assets/ui/js/tab.js#L67-L69

I just tested using href instead of data-tab-url and it works with no problems.  Anchor code used:
```
<a
<?= $tabs->linkable ?  'href="#'. $type . 'tab-' . str_slug($name) . '"' : 'href="#'. $type . 'tab-' . $index . '"' ?>
```

In my opinion, there is no need to preserve the tab url in the address bar when a tab is clicked.  Whether or not it should be preserved when navigated to directly is up for debate.

Reply:
Directly navigating is important for me. First situation: I have many tabs and on some of the tab is button to another part of backend. There shoud be something ended with "Save and return back" and better is returning to previous tab instead of first one. Second situation: Customer couldn't find something in backend. I can send him link to specific tab directly.

Reply:
@Rike-cz Totally agree.  Direct linking to tabs is important.  I was more commenting on whether the hashtag remains once the tab becomes active.  After thinking about it more from what you said, I think the hashtag should be preserved even after the tab becomes active.    I created a pull request a week ago without realizing this one existed to have linkable tabs, but it used history to remove the hashtag after the tab is selected.  Your PR is a lot simpler, as mine was completely in JS code adding a few more prototypes to tabs.js.  I definitely prefer your PR to mine.